__tests__/app.test.js

```diff
@@ -53,6 +53,7 @@ describe("app", () => {
             body: "some gifs",
             created_at: "2020-11-03T09:12:00.000Z",
             votes: 0,
+            comment_count: 2,
```

Comment:
single article shouldn't be returning an array. should be` {article:{....}}`

Reply:
also shouldn't amend previous tests for new features, add a new test ensuring the old one remain passing